### PR TITLE
chore(docs): add docs for releasing on android

### PR DESCRIPTION
**Pull request by jh2oman**
### Description

Docs outlining how to release mento to internal testers on android

### CODE REVIEW

File: RELEASES.md [hunk 1]
```diff
@@ -1,25 +1,10 @@
...
-
-## Why the unix time for version build codes?
-
-Using the unix time from the Git commit allows us to have a specific commit associated to each build we push and it's easy to lookup the commit from that value if needed:
-
-```
-git log --format='%H %ct' | grep TIMESTAMP
-```
+1. Run the [Release - Production Action](https://github.com/mobilestack-xyz/mobilestack-mento/actions/workflows/release-production.yml) against the `main` branch
+1. Download and unzip the android bundle which you can find in the build logs under android => fastlane-android (mainnet) => Upload Android build artifacts. [Here is an example](https://github.com/mobilestack-xyz/mobilestack-mento/actions/runs/11967777145/job/33365530323#step:16:51)
```
Comment (satish-ravi) [inline RELEASES.md]: IIRC, actions logs expire in 30 days so this could become a broken link

File: RELEASES.md [hunk 2]
```diff
@@ -1,25 +1,10 @@
...
+1. Navigate to [Internal Testing on the Google Play Console](https://play.google.com/console/u/0/developers/5695387721434163201/app/4974536396935190989/tracks/internal-testing) and click `Create new release`. Upload the app bundle that you just dowloaded and unzipped and click `Next`. Click `Save and Publish` on the next screen.
```
Comment (satish-ravi) [inline RELEASES.md]: seems like this link needs access? Should we add docs on how to get access?

File: RELEASES.md (hunk 1 above)
Comment (jh2oman) [reply RELEASES.md]: didn't know that! i'll remove the link

File: RELEASES.md (hunk 2 above)
Comment (jh2oman) [reply RELEASES.md]: Oh, do you not have access? I guess its only release owners

Comment (satish-ravi) [reply RELEASES.md]: yes, I do not.

**Review by satish-ravi — APPROVED** [review]

**Pull request merged**